beamer/deploy/commands.py

```diff
@@ -95,3 +96,101 @@ def deploy_base(
     path = artifacts_dir / "base.deployment.json"
     beamer.deploy.artifacts.generate(path, account.address, (resolver,))
     log.info("Generated artifact", path=str(path))
+
+
+@click.command("deploy")
+@click.option(
+    "--rpc-file",
+    "rpc_file",
```

Comment:
why do we need this here?

Reply:
We don't, removed both here and for deploy-base. Thanks.